modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaCommerceDestinationNamesCheck.testjava

```diff
@@ -0,0 +1,16 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+public class UpgradeJavaCommerceDestinationNamesCheck {
+
+	public static final String BASE_PRICE_LIST = DestinationNames.COMMERCE_BASE_PRICE_LIST;
```

Comment:
@michaelmdelcavalcanti this is a visual bug?

Reply:
Yes.